packages/web/src/app/doc-pages/components/css-library-doc/css-library-doc.component.html

```diff
@@ -3,6 +3,69 @@
   <div ngProjectAs="headerDescription" [innerHTML]="description"></div>
 </app-component-header>
 
+<app-component-section sectionTitle="Installation">
+  <app-component-subsection>
+    <ol class="e-list e-list--numbers">
+      <li>
+        <strong>Install package</strong>
+        <div class="e-mt-8">
+          <app-copy message="{{ npmInstall }}">
+            <code>{{ npmInstall }}</code>
+          </app-copy>
+          <span class="e-text-md e-mt-8 e-mb-8 e-mr-8 e-ml-8">or</span>
+          <app-copy message="{{ yarnAdd }}">
+            <code>{{ yarnAdd }}</code>
+          </app-copy>
+        </div>
+      </li>
+      <li>
+        <strong>Import stylesheet and script</strong>
+        <div class="e-mt-8">
+          <app-copy [message]="importStylesheet">
+            <code>{{ importStylesheet }}</code>
+          </app-copy>
+        </div>
+        <div class="e-mt-8">
+          <app-copy [message]="importScript">
+            <code>{{ importScript }}</code>
+          </app-copy>
```

Comment:
There is inconsistent use for passing props to `app-copy` here. The two above use `[message]={{ abc }}` while the two other use `[message]="abc"` 🤔